src/components/Pagination.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, Center } from "@chakra-ui/react";
+import React from "react";
+
+interface PaginationProps {
+  paginationPages: number;
+  activePage: number;
+  handlePaginationPageChange: (paginationPages: number) => void;
```

Comment:
I don't think you need to say `pagination` everywhere in the `Pagination` component. `handlePageChange: (page: number) => void;` should be enough